ACHNBrowserUI/ACHNBrowserUI/packages/Backend/Sources/Backend/environments/UserCollection.swift

```diff
@@ -215,8 +217,33 @@ public class UserCollection: ObservableObject {
         save()
     }
 
-    public func deleteDesign(at design: Int) {
-        designs.remove(at: design)
+    public func deleteDesign(at index: Int) {
+        designs.remove(at: index)
+        save()
+    }
+
+    // MARK: - Chores
+
+    public func addChore(_ task: Chore) {
+        chores.append(task)
+        save()
+    }
+
+    public func deleteChore(at index: Int) {
+        chores.remove(at: index)
+        save()
+    }
+
+    public func toggleChore(_ task: Chore) {
+        let index = chores.firstIndex(where: { $0.id == task.id })!
```

Comment:
I would rather avoid force unwrapping
```swift
guard let index = chores.firstIndex(where : { $0.id == task.id }) else { return }
```
and if I'm not wrong, as `Chore` is Hashable you could even simplify by
```swift 
guard let index = chores.firstIndex(of: task) else { return }
```

Reply:
Definitely. I missed that when reviewing my own code. Thanks!

Reply:
Done, @renaudjenny.